html/index.html

```diff
@@ -0,0 +1,32 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Tic Tac Toe</title>
+	<link rel="stylesheet" type="text/css" href="../css/style.css">
+</head>
+<body>
+<header>Tic Tac Toe</header>
+<center>
+<p id="gameStatus">Player 1, please make your move.</p>
+<div class="gameboard">	
+	<div class="row1">
+		<span class="winrow1 col1 diag1" id="square1"></span>
+		<span class="winrow1 col2" id="square2"></span>
+		<span class="winrow1 col3 diag2" id="square3"></span>
+	</div>
+	<div class="row2">
+		<span class="winrow2 col1" id="square4"></span>
+		<span class="winrow2 col2 diag1 diag2" id="square5"></span>
+		<span class="winrow2 col3" id="square6"></span>
+	</div>
+	<div class="row3">
+		<span class="winrow3 col1 diag2" id="square7"></span>
+		<span class="winrow3 col2" id="square8"></span>
+		<span class="winrow3 col3 diag1" id="square9"></span>
+	<div class="clear">
+		<button type="button" id="clear">Play Again!</button>
+	</div>
```

Comment:
Solid layout, and good naming of classes. We'll be seeing the `row` and `column` classes in a lot of CSS frameworks.